app/models/organisation_user.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: organisation_user
+#
+#  id              :integer          not null, primary key
+#  organisation_id :integer
+#  user_id         :integer
+#
+
+class OrganisationUser < ApplicationRecord
```

Comment:
If this is just a join table I don't think you need or want this model, it's just noise.